docs/schemas/itemRaw.json

```diff
@@ -0,0 +1,733 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "tas-answer",
+  "description": "An item anwser",
+  "definitions": {
+
+    "tas-file": {
+      "description": "Uploaded file",
+      "type": "object",
+      "properties": {
+        "data": {
+          "type": "string",
+          "media": {
+              "binaryEncoding": "base64"
+          }
+        },
+        "mimeType":   { "type": "string" },
+        "name":       { "type": "string" },
+        "via":        { "type": "string" },
+        "url":        { "type": "string" },
+        "confirmUrl": { "type": "string" }
+      }
+    },
+
+    "tas-stringAnswer": {
+      "description": "Schema for string answer",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Answer for this item",
+          "type": "string"
+        }
+      }
+    },
+
+    "tas-chronoAnswer": {
+      "description": "Schema for string answer",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Answer for this item",
+          "type": "string"
+        }
+      }
+    },
+
+    "tas-singleLineAnswer": {
```

Comment:
The examples don't validate without the type field, as the objects can fit multiple branches of the oneOf statement. Similarly, you can't deserialize the answers into the appropriate strongly typed objects unless there is something to tell the deserializer what type of object it's dealing with.